src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockDepths/instance_blackrock_depths.cpp

```diff
@@ -42,6 +44,7 @@ enum Creatures
     NPC_DOOMREL             = 9039,
     NPC_MAGMUS              = 9938,
     NPC_MOIRA               = 8929,
+    NPC_PRIESTESS_THAURISAN = 10076,
```

Comment:
`NPC_PRIESTESS_THAURISSAN`

(or `NPC_HIGH_PRIESTESS_OF_THAURISSAN` if you want to be precise)